cypress/support/commands.js

```diff
@@ -49,6 +49,9 @@ Cypress.Commands.add('login', (email, username, password) => {
 
 Cypress.Commands.add('createArticle', (title, description, body) => {
   cy.getCookie('auth').then((token) => {
+    if (!token) {
+      throw new Error('Auth cookie not found. Make sure you are logged in.');
+    }
```

Comment:
You're throwing an error if the auth cookie is not found. That's a good practice, but you might want to display a user-friendly message instead of throwing an error. This would improve the user experience.